scripts/etc/init.d/rcS

```diff
@@ -142,6 +143,15 @@ echo "================================================="
 echo "https://github.com/debian-pi/raspbian-ua-netinst/"
 echo "================================================="
 
+echo -n "Starting HWRNG "
+modprobe bcm2708-rng
+if [ $? -eq 0 ]; then
+    echo "succeeded!"
+else
+    echo "FAILED! (continuing to use the software RNG)"
+fi
+/usr/sbin/rngd -r /dev/hwrng
```

Comment:
I would put this line directly after the `modprobe` so that the check for success also takes the starting of the daemon into account